loop_rate:    1.0   # The frequency in Hz at which to run the control loops.
queue_size:   10    # The size of the message queue used for publishing and subscribing to topics.
create_map:   false # Whether to create a grid map from the coordinates.
cell_warn:    1000  # Number of grid cells in the map above which a performance warning is issued.
resolution:   1.0   # Resolution of the grid map representing the area in meter / cell.
